docs/semgrep-supply-chain/triage-remediation.md

```diff
@@ -107,6 +107,7 @@ Use filters to narrow down your results. The following criteria are available fo
 | **Status**             | Filter the triage state of a finding: <ul><li>**Open**: Findings for which there have been no triage or remediation action.</li><li>**Reviewing**: Findings that require more investigation to determine what the next steps should be.</li><li>**Fixing**: Findings for which you have decided to fix. Commonly used to indicate that these findings are tracked in Jira or assigned to developers for further work.</li><li>**Ignored**: Vulnerabilities that have been triaged as **Ignored** by the user. </li><li>**Fixed**: Vulnerabilities that are no longer detected after a scan. This typically means that the dependency containing the vulnerability has been updated. Semgrep Supply Chain automatically checks if the dependency has been updated and sets the vulnerability's status as **Fixed**.</li></ul>|
 | **Severity**           | Filter by the severity of a finding. Filters are based on the severity of a vulnerability. Semgrep Supply Chain rules use severity values set by the GitHub Advisory Database.  |
 | **Transitivity**       | Filter by the transitivity of the finding. <ul><li>**Direct**: Your project depends directly on the dependency.</li><li>**Transitive**: Your project's dependency depends on a vulnerable dependency.</li><li>**Undetermined**: Semgrep had no transitivity information for the dependency as it relates to your project.</li></ul> |
+| **EPSS probability**   | Exploit prediction scoring system (EPSS) probability. Represents the likelihood that the vulnerability will be exploited in the wild in the next 30 days. Values range from 0% to 100% where the higher the score, the greater the probability the vulnerability is exploited. Semgrep groups probabilities as follows: <ul><li><b>High</b>: 50 - 100%</li><li><b>Medium</b>: 10 - &#60;50%</li><li><b>Low</b>: &#60;10% </li></ul> |
```

Comment:
We should probably reference FIRST.org here to make sure it's clear that we didn't come up with these scores ourselves